src/rpc_api/data_types.rs

```diff
@@ -46,11 +46,15 @@ use libipld_core::ipld::Ipld;
 use libp2p::PeerId;
 use nonempty::NonEmpty;
 use num_bigint::BigInt;
+use once_cell::sync::Lazy;
 use parking_lot::RwLock as SyncRwLock;
 use serde::{de, Deserialize, Deserializer, Serialize, Serializer};
 use serde_json::Value;
 use tokio::sync::RwLock;
 
+// Define a static variable for the default PeerId
+static DEFAULT_PEER_ID: Lazy<PeerId> = Lazy::new(PeerId::random);
+
```

Comment:
Why do we need any randomness at all?

Reply:
we don't need randomness here
removed it